HOW/social/process/how/how-v1.md

```diff
@@ -0,0 +1,163 @@
+
+### HOW Format
+
+HOW standards live as markdown file named `how.md` at the leaves of this directory structure.  The format of a HOW standard file consists of a markdown header and a markdown body.  The content of the header and body MUST follow a template specified by:
+1.  merging the `_requirements.md` files that live at each level of the tree including and above the leaf where the HOW is being added
+2.  merging in any additional body sections or headers specified by a status change in a process flow template.
+
+The above format specification allows the templates for parts of the HOW tree to be contextually appropriate for that part of the tree.  For example, `technical/application` standards require a "Reference Implementation" section, what would not be appropriate for a `social/process` or `info` standard.
+
+### HOW Workflow
+
+HOWs transition through three stages: Define, Refine, & Align, each of which may have one or more statuses that formally indicate the steps in the workflow of a HOW for that stage. In the future we intend for there to be many flavors of the stages of this workflow, but this default HOW process specifies just one and the steps for each stage are as follows:
+
+#### Define
+This is the ideation stage, when a standard is first thought up and proposed.  The status for this stage is: `Draft`.  The steps in the stage are:
+
+1. Author(s) fork the HOW repository, and create a new HOW at the appropriate place in the HOW tree.  Author(s) MUST follow the template for that spot in the tree, as well as add this process (`/social/process/how/how-v1.md`) as the `Source` to the header of the HOW.
+2. Author(s) MAY discuss ideas in various Holochain community forums.
+3. Author(s) MAY get early feedback from core developers.
+4. This stage is formally complete when a pull request is made against the HOW repository.  The pull request must be made using the "Create HOW draft" template. Additionally, when submitting the pull request, Author(s) MUST check "allow edits and access to secrets by maintainers".
+
+#### Refine
+This is the review stage, when Author(s) are ready for feedback from the core developers and the community at large. The status for this stage are: `REVIEW`, `LAST CALL`.  The steps in this stage are:
+
+1. A member of the core team or requested working group will do an initial evaluation of the PR.  If the a team member of working group member is willing to take on stewardship of this HOW they will update the header status to `REVIEW` and will add the section(s) specified in `/social/process/how/_requirements.md#Process Template Section: REVIEW`.  If no requested steward is willing take on the HOW it MUST be updated to status `REJECTED` and the section(s) specified in `/social/process/how/_requirements.md#Process Template Section: REJECTED` added.
```

Comment:
I don't understand the point of setting it to rejected. I assume we don't want to merge every spammy pr that may be opened filling our directory structure with rejections. Shouldn't we just close the PR?

Reply:
So, this certainly makes sense for spam and other things that are cursorily rejected because they don't actually get taken up by the team, or a working-group or some steward to consider.  But if there were taken up and considered at length, and especially if there's some formal process to say "nope we actually don't want to do this", it seems like it's worth holding on to that information inside the HOW process not just in github close PR comment.

But maybe we could have that also fall under the `ABANDONDED` status with a `Reason` that would be "Formally rejected" for that case?